python_tests/python/build_and_run.bat

```diff
@@ -0,0 +1,3 @@
+cargo build
+copy ..\..\target\debug\rust_swig_test_python.dll rust_swig_test_python.pyd
```

Comment:
hardcoded path to target is bad, and why we need this script,
there is ci_build_and_test.py ?